packages/grafana-ui/src/components/Table/TableNG/TableNG.tsx

```diff
@@ -187,7 +187,7 @@ export function TableNG(props: TableNGProps) {
     }
   };
 
-  const mapFrameToDataGrid = (main: DataFrame) => {
+  const mapFrameToDataGrid = (main: DataFrame, rows: TableRow[]) => {
```

Comment:
I think [Ihor's filter PR also requires the rows dependency](https://github.com/grafana/grafana/pull/96767/files#diff-52a492a6ef58360ea7e6f938c21e556c66b4940d8c8d0483b1f3cf319ca28419) in this function. I guess it was inevitable to have to re-order this stuff.